test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/cluster/ClusterUtils.java

```diff
@@ -55,25 +57,57 @@
 public class ClusterUtils {
   private static final AtomicInteger keyspaceId = new AtomicInteger();
 
+  private static final String CLUSTER_BUILDER_CLASS =
+      System.getProperty(
+          "cluster.builder",
+          "com.datastax.oss.driver.api.testinfra.cluster.DefaultClusterBuilderInstantiator");
+
+  @SuppressWarnings("unchecked")
+  public static <T extends CqlSession> ClusterBuilder<?, ? extends Cluster<T>> baseBuilder() {
+    try {
+      Class<?> clazz = Class.forName(CLUSTER_BUILDER_CLASS);
+      Method m = clazz.getMethod("builder");
+      return (ClusterBuilder<?, ? extends Cluster<T>>) m.invoke(null);
+    } catch (Exception e) {
+      e.printStackTrace();
+      // TODO: Logger
```

Comment:
Is it just a matter of declaring an SLF4J logger?

Reply:
Err woops, yes, it's pretty simple just forgot to do it, will fix.